src/cloud-api-adaptor/test/utils/checkout_kbs.sh

```diff
@@ -27,8 +27,37 @@ pushd "${TEST_DIR}/trustee"
 git checkout "${KBS_VERSION}"
 KBS_SHA="$(git rev-parse HEAD)"
 
-install_kbs_client "${KBS_SHA}"
+YAML_FILE="kbs/config/kubernetes/base/deployment.yaml"
+
+yq -i '.spec.template.spec.containers[0].volumeMounts += [{"name": "kbs-https-certificate", "mountPath": "/etc/kbs1"}, {"name": "kbs-https-key", "mountPath": "/etc/kbs2"}]' "$YAML_FILE"
```

Comment:
Are we able to use `kustomize` to update these, rather than yq/sed which might be more brittle?